src/time.rs

```diff
@@ -60,3 +55,32 @@ impl Time {
     /// used).
     pub fn from_seconds_since_unix_epoch(secs: u64) -> Time { Time(secs) }
 }
+
+#[cfg(feature = "std")]
+impl std::convert::TryFrom<::std::time::SystemTime> for Time {
+    type Error = ring::error::Unspecified;
```

Comment:
The use of `ring::error::Unspecified` in the original `try_from` function's implementation was a mistake. You don't need to change it in this PR since it was already wrong.